src/test/java/com/google/sps/CommentAnalysisTest.java

```diff
@@ -44,105 +47,110 @@
 /** This is a JUnit test for sentiment mockedAnalysis */
 @RunWith(JUnit4.class)
 public class CommentAnalysisTest {
+  private static final double LOWER_END_VAL = -1.0;
+  private static final double UPPER_END_VAL = 1.0;
+  private static final BigDecimal INTERVAL = BigDecimal.valueOf(0.2);
+  private static final BigDecimal UPPER_END = BigDecimal.valueOf(UPPER_END_VAL);
+  private static final BigDecimal LOWER_END = BigDecimal.valueOf(LOWER_END_VAL);
   private static final float TEST_SCORE = 0.23f;
-  private static Comment testTopComment = new Comment();
-  private static CommentSnippet topCommentSnippet = new CommentSnippet();
-  private static CommentThread testCommentThread = new CommentThread();
-  private static List<CommentThread> testCommentThreadList =
-      new ArrayList<>(Arrays.asList(testCommentThread, testCommentThread));
-  private static CommentThreadListResponse youtubeResponse = new CommentThreadListResponse();
-  private static CommentThreadSnippet testThreadSnippet = new CommentThreadSnippet();
-  private static LanguageServiceClient mockedlanguageService =
-      mock(LanguageServiceClient.class, Mockito.RETURNS_DEEP_STUBS);
-  private static CommentAnalysis commentAnalysis = new CommentAnalysis(mockedlanguageService);
 
-  @Before
-  public void setUp() {
-    topCommentSnippet.setTextDisplay("Test Message");
-    testTopComment.setSnippet(topCommentSnippet);
-    testThreadSnippet.setTopLevelComment(testTopComment);
-    testCommentThread.setSnippet(testThreadSnippet);
-    youtubeResponse.setItems(testCommentThreadList);
+  /**
+   * It constructs a HashMap with current range for expected score categorizations for comparisions.
+   *
+   * @param frequency a list of frequency corresponding to each interval sorted in ascending order
+   * @return constructed hashmap with keys as ranges based on intervals and values corresponding to
+   *     frequency input
+   */
+  private Map<Range, Integer> initializeMap(List<Integer> frequency) {
```

Comment:
Maybe rename to something like `constructRangeMapFromFrequencyList()`.

Without looking at the actual method definition, it's difficult to determine the intention of `initializeMap`.